h2o-py/tests/testdir_algos/gbm/pyunit_gbm_wendy_problem.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division
+from builtins import range
+import sys
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils, assert_equals
+from h2o.estimators import H2OGradientBoostingEstimator, H2OXGBoostEstimator
+from h2o.tree import H2OTree
+
+
+def assert_list_equals(expected, actual, delta=0):
+    assert_equals(len(expected), len(actual))
+    for i in range(0, len(expected)):
+        assert_equals(expected[i], actual[i], delta=delta)
+
+
+def gbm_predict_contributions_sorting_large():
+    fr = h2o.import_file(pyunit_utils.locate("bigdata/laptop/covtype/covtype.full.csv"))
+    # build first model with one hardware configuration    
+    ntrees=50
+    for i in range(1):
+        m = H2OGradientBoostingEstimator(seed=1234, ntrees=ntrees)
+        m.train(x=list(range(0, 12)), y="Cover_Type", training_frame=fr)
+        # pred = m.predict(fr)
+        #h2o.download_csv(pred, "/some/dictory/pred.csv"),         # build second model with same hardware configuration    
+        m2 = H2OGradientBoostingEstimator(seed=1234, ntrees=ntrees)
+        m2.train(x=list(range(0, 12)), y="Cover_Type", training_frame=fr)
+        # pred2 = m2.predict(fr)
+        #h2o.download_csv(pred2, "/some/directory/pred2.csv")        # compare prediction frames should be the same.  Basically, save the prediction as a csv and load it and compare    
+        # pred = h2o.import_file("/path/to/pred.csv")    
+        # pred2 = h2o.import_file("/path/to/pred2.csv")    
+        # for index in range(1, pred.ncols):
+        #     compare_frame_one_column(pred[index], pred2[index])
+    
+        # compare variable importance should be the same.  Can just print it out too if the two models are from different cluster runs    
+        relative_importance = m.varimp()
+        print(relative_importance)
+        relative_importance2 = m2.varimp()
+        print(relative_importance2)
```

Comment:
Since we don't care about relative_importance here, can you remove all codes pertaining to it?

Reply:
I have also covered PUBDEV-9000 bug here

Reply:
I mean only in test, the bug is still there